lib/npm/tagger.js

```diff
@@ -61,24 +82,13 @@ Tagger.prototype.delete = function (opts, callback) {
  * @returns {undefined}
  */
 Tagger.prototype.add = function (opts, callback) {
-  var pkg = opts.pkg,
-    tag = opts.tag,
-    version = opts.version,
-    Package = this.Package;
-
-  //
-  // Create update body for cassandra
```

Comment:
logic has been moved to `_createBody` to allow `delete` and `add` to profit from it